x/escrow/init.go

```diff
@@ -20,7 +22,7 @@ func (i *Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
 		Sender    weave.Address   `json:"sender"`
 		Arbiter   weave.Condition `json:"arbiter"`
 		Recipient weave.Address   `json:"recipient"`
-		Timeout   int64           `json:"timeout"`
+		Timeout   time.Time       `json:"timeout"`
```

Comment:
Curious as to default parsing logic for time.Time... I guess it is the iso time below. Does it have to be in UTC?
What happens if we use a non UTC time? Maybe we should force conversion to UTC after parsing.

A few tests with parsing and behavior of non-utc time would be interesting.

Reply:
This is working (most of the time) as expected :joy_cat: 

I think two equal time values in different time zones are equal. You can have different timezone from parsing as well for example by using an offset `-0700`.

The issue is sometimes in tests when you use `reflect.DeepEqual` - having timezone difference makes two times not equal. This is an issue hard to spot. 

I will add the tests.